docs/running_libE.rst

```diff
@@ -66,8 +66,7 @@ where ``N`` is the number of workers. This will launch one manager and
 
 libEnsemble will run on one node in this scenario. It is only suitable for running
 in :doc:`central mode<platforms/platforms_index>` on multi-node systems. It can also
-be used on stand-alone systems. *Technically, you could run without dedicated_mode set,
```

Comment:
Why remove? 
I think the central v distributed was a bit of bad distinction. Some users, including WarpX/FBPIC users run libEnsmeble on one node and share with launched applications, while other nodes will just run launched applications. The first node does not conflict, because their WarpX runs are limited by number of GPUs (with one cpu per gpu), so there is plenty to run libEnsemble on that node. This means that my original line that its only suitable for 'central mode', given our definition, is not true. So I think we could re-word better. 
Dedicated v shared for libEnsemble might ultimately be better words.